bundles/org.openhab.core.id/src/main/java/org/openhab/core/id/InstanceUUID.java

```diff
@@ -75,19 +73,21 @@ public static synchronized String get() {
     private static void writeFile(File file, String content) throws IOException {
         // create intermediary directories
         file.getParentFile().mkdirs();
-        try (OutputStream outputStream = new FileOutputStream(file)) {
-            IOUtils.write(content, outputStream);
+        try {
+            Files.writeString(file.toPath(), content, StandardCharsets.UTF_8);
+        } catch (IOException ioe) {
+            LOGGER.warn("Failed writing to file '{}': {}", content, file.getAbsolutePath(), ioe.getMessage());
         }
     }
 
     private static String readFirstLine(File file) {
-        List<String> lines = null;
         try {
-            lines = IOUtils.readLines(new FileInputStream(file));
+            final List<String> lines = Files.readAllLines(file.toPath(), StandardCharsets.UTF_8);
+            return lines.isEmpty() ? "" : lines.get(0);
```

Comment:
We only need the first line. Could be changed to:

```suggestion
            final BufferedReader reader = Files.newBufferedReader(path);
            return reader.readLine();
```